components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -4435,20 +4444,20 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
                 attributionId: req.attributionId,
             });
         }
-        const user = await this.checkAndBlockUser("listUsage");
-        await this.guardCostCenterAccess(ctx, user.id, attributionId, "get");
+        await this.checkAndBlockUser("listUsage");
+        await this.guardCostCenterAccess(ctx, attributionId, "get", "not_implemented");
```

Comment:
I would have expected this to be `read_billing` after reading the model. Intentional?

Reply:
Leaving it up to you to follow up on this. :+1: